web/sites/default/config/core.entity_form_display.node.location.default.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
   config:
     - field.field.node.location.field_address_or_entrance
     - field.field.node.location.field_geolocation
+    - field.field.node.location.field_group
```

Comment:
Well shoot. This is not going to work. If location is not first added to the groups via content plugin then this new field will cause a ton of issues. 🤔 

For now, let's remove the requirement of the group field on location. I want to give that some more thought.